docs/model_developers/modules.rst

```diff
@@ -347,10 +347,26 @@ Add the following content to the file:
   login <username>
   password <password>
 
-For more information see the doc about`pip authentication <https://pip.pypa.io/en/stable/topics/authentication/>`_.
+For more information see the doc about `pip authentication <https://pip.pypa.io/en/stable/topics/authentication/>`_.
 
 You will also need to specify the url of the repository in the ``project.yml`` file of your project (See: :ref:`specify_location_pip`).
 
 By following the previous steps, the Inmanta server will be able to install modules from a private Python package repository.
 
 
+Inter-module dependencies
+#########################
+
+The plugins code of a module A can have a dependency on the plugins code of another V2 module B. When doing this,
+care should be taken that the Python module(s) you depend on, do not define any resources or providers. Otherwise the
```

Comment:
Have you checked this with the solutions team? From the ticket:

> technically it can iff they are always there but that's more difficult to explain so only mention that if the solutions team sees a use case for it

Another, similar, technicality that we probably want to skip: the module can contain resources as long as those resources are not present in the model.

Reply:
> Have you checked this with the solutions team?

Yes.